apps/backend/src/usecases/owner/post-pet-for-adoption/post-pet-for-adoption.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { INestApplication } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import {
+  closeInMongodConnection,
+  TestMongoDataServicesModule,
+} from '../../../../test/utils/in-memory-mongo';
```

Comment:
What do you think about using path aliases and barrel files to simplify and better organize imports?

Reply:
yep, that's a good idea! just haven't had the time yet, but feel free to contribute :D